app/components/pilas-blockly.js

```diff
@@ -6,7 +6,7 @@ let VERSION_DEL_FORMATO_DE_ARCHIVO = 1;
 export default Ember.Component.extend({
   classNames: 'desafio-panel-derecho',
   ejecutando: false,
-  debeReiniciar: false,
+  terminoDeEjecutar: false,
```

Comment:
Che, ¿Cuál es la diferencia entre "ejecutando" y "terminó de ejecutar"?

Reply:
Que _ejecutando_ es el flag que se activa mientras se está ejecutando el código y _terminoDeEjecutar_ el que se activa cuando el código terminó de ejecutar (que queda en el estado final y aún así se debería reiniciar).

Reply:
Ok, me había parecido q uno era el not del otro.

Reply:
Nop, empiezan los dos en falsos, luego solo ejecutando true, luego solo terminoDeEjecutar true. Y al reiniciar se repite.